_layouts/news.html

```diff
@@ -9,13 +9,13 @@
  <div class="col">
   <h1>
     News about
-    <a href="/">seL4</a>
+    <a href={{ "/" | relative_url }}>seL4</a>
     and the
-    <a href="/Foundation">seL4 Foundation</a>
+    <a href={{ "/Foundation" | relative_url }}>seL4 Foundation</a>
   </h1>
  </div>
  <div class="col">
-Older News:  {% for i in (2020..newest_year) -%}<a href="/news/{{ i }}">{{ i }}</a> {% endfor %}
+Older News:  {% for i in (2020..newest_year) -%}{% assign url = "/news/" | relative_url %}<a href="{{ url }}{{ i }}">{{ i }}</a> {% endfor %}
```

Comment:
I'd put the `{%- assign url = "/news/" | relative_url -%}` on its own line outside of the for-loop (but with the `-` to trim the added whitespace/newlines around it).

Just curious, but does `{{ url i }} also work?

Reply:
For `{{ url i }}` it gives me a syntax error.